roadmap.md

```diff
@@ -25,22 +24,24 @@ Architecture:
 
 Active ports:
 
-- Languages: F# (C# and VB are done).
-- OSes: Linux, OS X, FreeBSD and Windows.
+- Languages: C#, VB, and F#.
+- OS: Linux, macOS, FreeBSD and Windows.
 - Linux distros: RHEL, Debian family, CentOS.
```

Comment:
RHEL, CentOS, Fedora, Ubuntu 14.04, Ubuntu 16.04, Debian 8.2, Linux Mint. You need to put them all in. :)